src/main/java/edu/ucsb/cs56/mapache_search/AuthControllerAdvice.java

```diff
@@ -38,6 +38,7 @@ public String getUid(OAuth2AuthenticationToken token){
             AppUser u = new AppUser();
             u.setUid(uid);
             u.setUsername(token2login(token));
+            u.setMaxsearches(20l);
```

Comment:
This "magic number" should probably be a 

```java
public final static long MAX_SEARCHES = 20;
```

or something that, accompanied by a comment that has the URL of where that number comes from.
And is it 20 ?   I thought it was more like 100 or something...

Reply:
done